src/django_perf_rec/operation.py

```diff
@@ -4,15 +4,17 @@
 
 
 class Operation:
-    def __init__(self, alias, query):
+    def __init__(self, alias, query, trace_back):
         self.alias = alias
         self.query = query
+        self.tb = trace_back
```

Comment:
I changed the internal name to `traceback` since there's no reason for it to differ